tv-shows/src/components/features/review/ReviewItem/ReviewItem.tsx

```diff
@@ -1,23 +1,41 @@
 'use client';
 
 import { Avatar, Card, CardBody, CardFooter, CardHeader, Flex, Text } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { mutate } from 'swr';
+import useSWRMutation from 'swr/mutation';
+import { AUTH_LOCAL_STORAGE_KEY } from '../../../../constants';
+import { deleteReview } from '../../../../fetchers/reviews';
+import { swrKeys } from '../../../../fetchers/swrKeys';
 import { IReview } from '../../../../typings/Review.type';
+import { ILocalStorageAuth, loadFromLocalStorage } from '../../../../utils/localstorage-helpers';
 import { StyledButton } from '../../../core/StyledButton/StyledButton';
 import { StarsRating } from '../../../shared/StarsRating/StarsRating';
+import { EditReviewModal } from '../EditReviewModal/EditReviewModal';
 
 interface IReviewItemProps {
 	review: IReview;
-	onRemoveReview: (uuid: string) => void;
 }
 
-export const ReviewItem = ({ review, onRemoveReview }: IReviewItemProps) => {
+export const ReviewItem = ({ review }: IReviewItemProps) => {
+	const [userID, setUserID] = useState<string>();
```

Comment:
Is this the same ID you get from useSwr(user)? If yes, why not use that instead of managing local storage state?

Reply:
Fixed